### DISCUSSION TITLE
How to enable 802.11ac mode (default chipset) FAQ broken 404

### DISCUSSION THREAD

**Discussion by smanley** [RaspAP/raspap-webgui#936] [Q&A] [answered]
The FAQ on the website is no longer found. https://docs.raspap.com/faq/#80211ac

Can someone let me know how to enable this option for the GUI?

Thanks,
Steve

**Comment by smanley** [comment, marked as answer]
https://docs.raspap.com/faq.html#why-is-the-80211ac-5ghz-wireless-mode-option-disabled-in-configure-hotspot

N/M figured it out. You may wish to fix the broken link.

**Reply by billz** [reply]
There's been an update to the site generation software for our docs that has apparently broken these permalinks. Will look into it. Thanks for the heads up.

**Reply by billz** [reply]
All good now. Much appreciated, Steve
